templates/blink/intent_to_implement.html

```diff
@@ -1,31 +1,3 @@
-<p>Email to</p>
```

Comment:
Since we are deleting this, we just need to make sure that all 3 of your changes go out in the same deployment.  It should be OK if a user has a browser tab open on a previous version of the app because as soon as the click the button it will open a new tab that does a full page load and gets the latest version of the SPA.

Reply:
Actually, this PR should be available to deploy on its own, as this piece of the intent page has been moved to `launch.html`. The existing page should look exactly the same, but this `intent_to_implement.html` only contains the template for intent body contents, and no aspects of the surrounding intent draft preview page. 🙂

Reply:
Ah, OK.